$OpenBSD$

This is displayed on the 'a' page. Seems that if this (or other
bytes/packets divisions) are not an exact integer some far-too-high
value is printed e.g.

│ Packets: 6680
│ Bytes:   737.5k (755203)
│ Average: ~1089172957 B/Pkt

│ TYPE    Packets    Bytes ~B/P  Pkts% Byte% Usage%
│ BEACON      188    30.1k 10737470.7  70.1  70.1
│ BACK          7      224   32  2.6   0.0   0.0
│ PSPOLL        6      120   20  2.3   0.0   0.0
│ QDNULL       40  4095.9M 21474815.0  29.9  29.9
│ ACK          25      350   14  9.4   0.0   0.0
│ QDATA        11      173 1952250.27  0.3   0.3

224/7 = 32, 120/6 = 20, etc. -> integer -> b/p is ok
755203/6680 = 113.0543, 173/11 = 15.7272 -> not integer -> b/p is bogus

No idea why this isn't just truncated..

QDNULL high value is probably -1


Index: display-statistics.c
--- display-statistics.c.orig
+++ display-statistics.c
@@ -54,7 +54,7 @@ void update_statistics_win(WINDOW *win)
 	mvwprintw(win, 2, 2, "Packets: %d", stats.packets );
 	mvwprintw(win, 3, 2, "Bytes:   %s (%d)",
 		  kilo_mega_ize(stats.bytes), stats.bytes );
-	mvwprintw(win, 4, 2, "Average: ~%d B/Pkt", stats.bytes / stats.packets);
+	mvwprintw(win, 4, 2, "Average: ~%d B/Pkt", stats.bytes / stats.packets); /* XXX */
 
 	mvwprintw(win, 2, 40, "Retries:       %3.1f%% (%d)",
 		  stats.retries * 100.0 / stats.packets, stats.retries);
